### PR TITLE
ENG-3692 amend svc logs command

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: test/commands/svc/logs.test.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      )
+    })
+
+  test
+    .stderr()
+    .command(['svc logs'])
+    .catch(error => {
+      expect(error.message).to.contain('At least one service name is required.')
+    })
+    .it('restart without any arguments and flags')
```
Comment (zonia3000) [inline test/commands/svc/logs.test.ts]: restart -> logs

File: test/commands/svc/logs.test.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+  test
+    .stderr()
+    .command(['svc logs', 'win2000'])
+    .catch(error => {
+      expect(error.message).to.contain('Service win2000 is not enabled.')
+    })
+    .it('restart an unlisted service')
```
Comment (zonia3000) [inline test/commands/svc/logs.test.ts]: restart -> logs

File: test/commands/svc/logs.test.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  test
+    .stderr()
+    .stub(SvcService.prototype, 'logServices', sinon.stub().resolves(403))
+    .command(['svc logs', 'rabbitmq'])
+    .catch(error => {
+      expect(error.message).to.contain(
+        'Logs display service(s) rabbitmq failed, exited with code 403'
+      )
+    })
+    .it('restart command unsuccessful - exits with error code', () => {
```
Comment (zonia3000) [inline test/commands/svc/logs.test.ts]: restart -> logs

File: test/commands/svc/logs.test.ts (hunk 1 above)
Comment (jeffgo10) [reply test/commands/svc/logs.test.ts]: done

File: test/commands/svc/logs.test.ts (hunk 2 above)
Comment (jeffgo10) [reply test/commands/svc/logs.test.ts]: done

File: test/commands/svc/logs.test.ts (hunk 3 above)
Comment (jeffgo10) [reply test/commands/svc/logs.test.ts]: done

File: src/commands/svc/logs.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import { CliUx, Flags } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+import { SvcProcessResult, ServiceTypes } from './svc-process-result'
+
+export default class Logs extends SvcProcessResult {
+  static strict = false
+  static description = 'Display logs of running auxiliary services'
```
Comment (cecchisandrone) [inline src/commands/svc/logs.ts]: Display running auxiliary services logs

Comment (jeffgo10) [reply src/commands/svc/logs.ts]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**